src/Nethermind/Nethermind.JsonRpc/WebSockets/JsonRpcSocketsClient.cs

```diff
@@ -70,15 +71,15 @@ public async Task ReceiveAsync(Memory<byte> data)
             {
                 _jsonRpcLocalStats.ReportCall(result.Report, handlingTimeMicroseconds, size);
             }
-            
+
             Interlocked.Add(ref Metrics.JsonRpcBytesSentWebSockets, data.Length);
         }
 
         public async Task<int> SendJsonRpcResult(JsonRpcResult result)
         {
             string SerializeTimeoutException()
             {
-                JsonRpcErrorResponse error = _jsonRpcService.GetErrorResponse(ErrorCodes.Timeout, "Request was canceled due to enabled timeout.");
+                JsonRpcErrorResponse error = _jsonRpcProcessor.JsonRpcService.GetErrorResponse(ErrorCodes.Timeout, "Request was canceled due to enabled timeout.");
```

Comment:
Law of Demeter violation. Any reason for not passing the implementations in the constructor?

Reply:
Removed